domain/flows/order_pizza.yml

```diff
@@ -0,0 +1,149 @@
+version: "3.1"
+
+intents:
+  - greet
+  - inform_pizza
+  - inform_address
+  - inform_num_pizza
+  - ask_availability
+  - correct_order
+  - correct_num_pizza
+  - correct_pizza_type
+  - correct_payment_option
+  - affirm
+  - deny
+
+entities:
+  - pizza_type
+  - address
+  - department
+
+slots:
+  pizza:
+    type: text
+    mappings:
+        - type: from_entity
+          entity: pizza_type
+  num_pizza:
+    type: float
+    mappings:
+        - type: from_entity
+          entity: number  # this is defined in the restaurant.yml domain file
+          intent: inform_num_pizza
+          conditions:
+            - active_flow: fill_pizza_order
+        - type: from_entity
+          entity: number
+          intent: correct_num_pizza
+          conditions:
+          - active_flow: fill_pizza_order
+          - active_flow: order_pizza
+  address:
+    type: text
+    mappings:
+      - type: from_text
+        intent: inform_address
+        conditions:
+          - active_flow: fill_pizza_order
+  confirmation_order:
+    type: bool
+    mappings:
+        - type: from_intent
+          intent: affirm
+          value: true
+          conditions:
+            - active_flow: fill_pizza_order
+        - type: from_intent
+          intent: deny
+          value: false
+          conditions:
+            - active_flow: fill_pizza_order
+  token:
+    type: text
+    mappings:
+      - type: from_llm
+        conditions:
+        - active_flow: auth_user
+  membership_points:
+    type: float
+    mappings:
+      - type: custom
+        action: action_check_points
+  payment_option:
+    type: categorical
+    values:
+      - card
+      - "membership points"
```

Comment:
Don't think `"` is needed.

Reply:
Updated: https://github.com/RasaHQ/rasa-calm-demo/pull/30/commits/e7001fc9a9d963468e3845fd51afc96a36f0898b